src/ocean/io/Stdout.d

```diff
@@ -59,10 +59,10 @@ static this ( )
 {
     // note that a static-ctor inside Layout fails
     // to be invoked before this is executed (bug)
```

Comment:
is comment still applicable? i would expect not, if new code works

Reply:
The only static ctor I could find is here: https://github.com/sociomantic-tsunami/ocean/blob/edf92dc7eea6869b8e54d99fe11f9a452a8ab1fe/src/ocean/text/convert/Layout_tango.d#L1033-L1036
I'll remove the comment